src/app/core/guards/onboarding.guard.ts

```
@@ -1,14 +1,19 @@
 import { Injectable } from '@angular/core';
 import { CanActivate, Router, UrlTree } from '@angular/router';
-import { Observable, of } from 'rxjs';
-import { catchError, map } from 'rxjs/operators';
+import { from, Observable, of } from 'rxjs';
```

ESLint

[error] 'from' is defined but never used.

(@typescript-eslint/no-unused-vars)